src/Products/MailHost/tests/testMailHost.py

```diff
@@ -692,6 +712,25 @@ def testStartQueueProcessorThread(self):
         md = zope.sendmail.maildir.Maildir(self.smtp_queue_directory)
         self.assertEqual(len(list(md)), 1)
 
+    @unittest.skipUnless(PY36, 'requires Python 3.6+')
+    def test_8bit_special(self):
+        mh = self._makeOne('MailHost')
+        mh.send(u"""\
+Mime-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+\u00e4
+""",
+                mto='user@example.com',
+                mfrom='zope@example.com',
+                subject='Hello world',
+                charset='utf-8')
+        transaction.commit()
+        self.assertTrue(mh.started_queue_processor_thread)
+        md = zope.sendmail.maildir.Maildir(self.smtp_queue_directory)
+        self.assertEqual(len(list(md)), 1)
+
```

Comment:
Just for my understanding. Why did you test `8bit` encoding in the `TestMailHostQueuing` test class and not in `TestMailHost` class?